rest/adminapitest/admin_api_test.go

```diff
@@ -2141,6 +2141,46 @@ func TestHandleCreateDB(t *testing.T) {
 	rest.RequireStatus(t, resp, http.StatusBadRequest)
 }
 
+func TestHandleCreateDBNoJsonName(t *testing.T) {
+	rt := rest.NewRestTester(t, nil)
+	defer rt.Close()
+
+	bucket := "albums"
+	kvTLSPort := 11207
+	resource := fmt.Sprintf("/%s/", bucket)
+
+	bucketConfig := rest.BucketConfig{Bucket: &bucket, KvTLSPort: kvTLSPort}
+	dbConfig := &rest.DbConfig{BucketConfig: bucketConfig, SGReplicateEnabled: base.BoolPtr(false)}
+
+	reqBody, err := base.JSONMarshal(dbConfig)
+	assert.NoError(t, err, "Error unmarshalling changes response")
+
+	resp := rt.SendAdminRequest(http.MethodPut, resource, string(reqBody))
+	rest.RequireStatus(t, resp, http.StatusCreated)
+	resp = rt.SendAdminRequest(http.MethodGet, resource, string(reqBody))
+	assert.Contains(t, resp.Body.String(), "\"db_name\":\"albums\"")
```

Comment:
Have a look at `TestServerlessFetchConfigsLimited` to see how to make a database on the RestTester through the rest API with ease. You should not need to provide a kv port, and bucket config.

By default, the RestTester creates a database called `db`, so it would be good to run the rest tester in persistent config mode (an option in the RT config) as this makes the rest tester not automatically create a database. The test will need to be skipped when running with Walrus.